### DISCUSSION TITLE
FingerPaint v1.2.6

### DISCUSSION THREAD

**Discussion by Wazzaps** [Wazzaps/fingerpaint#23] [Announcements]
## Bugfixes:
- Add warning if `XDG_SESSION_TYPE` is not specified

## Misc:
- Added Nix flake

**Full Changelog**: https://github.com/Wazzaps/fingerpaint/compare/v1.2.5...v1.2.6

<hr /><em>This discussion was created from the release <a href='https://github.com/Wazzaps/fingerpaint/releases/tag/v1.2.6'>v1.2.6</a>.</em>

**Comment by jhossbach** [comment]
Updated in the AUR.
